inclusive-design/wecount.inclusivedesign.ca
# Remove tools ourdata from nav

## Description
Removed Tools and Our Data links from the top navigation bar


## Steps to test

1. run `npm run dev`
2. make sure tools and our data are no longer in the navigation bar
3. make sure rest of site still works

**Expected behavior:** 

The site should work the same with the exemption of the tools and out data links in the top navigation bar

## Additional information



## Related issues



resolves #140
